deploy/grant-cluster-admin.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubepoc:cluster-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: apiserver
+  namespace: codeready-devconsole
```

Comment:
Let's use the `openshift-devops-console` namespace ( along the lines of `openshift-web-console` )

Reply:
Done.